DEFINITION.md

```diff
@@ -72,3 +72,11 @@ Tecnologias nativas ao cloud empoderam empresas a criarem e rodarem aplicações
 Essas técnicas permitem criar sistemas de baixo acoplamento, resilientes, gerenciáveis e observáveis. Combinadas com automações robustas, elas permitem que os engenheiros façam alterações de alto impacto de forma frequente e previsível, com o mínimo de esforço.
 
 A Cloud Native Computing Foundation procura conduzir a adoção desse paradigma auxiliando e sustentando um ecosistema de projetos de código aberto e não atrelados a nenhum fornecedor. Nós democratizamos padrões estado-da-arte para fazer com que essas inovações sejam acessíveis a todos.
+
+## Русский:
+
+Нативные облачные технологии позволяют организациям создавать и запускать масштабируемые приложения в современных динамических средах, таких как публичные, частные и гибридные облака. Контейнеры, service meshes, микросервисы, неизменяемая инфраструктура и декларативные API являются примером такого подхода.
```

Comment:
"Cloud native" instead of "Нативные облачные"? Not sure here. 
Also "service mesh проекты" vs "service meshes", what do you think?

Reply:
I've also thought regarding naming for Cloud Native, we could leave it as is as for other languages or for example add "Нативные облачные (Cloud native)". 

> A service mesh is a configurable, low‑latency infrastructure layer designed to handle a high volume of network‑based interprocess communication among application infrastructure services using application programming interfaces (APIs)

So here more about infrastructure rather than projects, I think. 

What do think?

Reply:
+1 on "Нативные облачные (Cloud native)".

Service Mesh already has a translation, tokened by @antweiss – сервисное сито. I think we can do the same: "сервисные сита (service meshes).

Reply:
"сервисное сито" sounds great, but so far has only 25 results in Yandex: https://yandex.ru/search/?text=%22%D1%81%D0%B5%D1%80%D0%B2%D0%B8%D1%81%D0%BD%D0%BE%D0%B5%20%D1%81%D0%B8%D1%82%D0%BE%22&lr=21772 

+1 to "сервисные сита (service meshes)" as a reasonable compromise.

Reply:
I think сервисные сети is more accurate translation.

Reply:
Well, my reasoning was: сеть is already used to translate 'network' -  so we need something else. Сито seems to be the perfect analogy for proxying all traffic.